packages/app/src/debug/DebugFailedTest.cy.tsx

```diff
@@ -1,5 +1,5 @@
 import DebugFailedTest from './DebugFailedTest.vue'
-import type { TestResults } from './DebugSpec.vue'
```

Comment:
This showed as `any` when importing types from a vue file into a tsx file so I moved them to another file to get better type safety.

Reply:
I am not opposed to moving the types to their own file in this instance when they are used multiple places.  There does seem to be a "right" way to export things from a Vue component  that uses the composition api.  You would use a separate normal `<script>` tag.  See here: https://vuejs.org/api/sfc-script-setup.html#usage-alongside-normal-script

Reply:
I'm going to revert these changes, I gave @astone123 suggestion a go following the docs for [enabling Volar take-over mode](https://vuejs.org/guide/typescript/overview.html#volar-takeover-mode) and my IDE is a lot happier.